mateo-mustapic/assignments/liferay-app/sdk/portlets/todo-portlet/docroot/view.jsp

```diff
@@ -25,12 +25,18 @@
 		<i class="icon-check"></i>
 	</h3>
 
+	<portlet:resourceURL id="task-resource" var="resourceUrl" />
+
 	<aui:form cssClass="task-form" name="task-form">
 		<aui:input cssClass="input-item" name="task" placeholder="enter-task" required="true" />
 
 		<aui:button cssClass="add-task" name="add" type="submit" value="add" />
 	</aui:form>
 
+	<div id="<portlet:namespace />request-success"></div>
+
+	<div id="<portlet:namespace />request-failure"></div>
```

Comment:
There is no need for two ```div```s, for success or failure, just have one ```div``` for request result status.